java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -194,6 +194,18 @@
   ORC_MAX_DISK_RANGE_CHUNK_LIMIT("orc.max.disk.range.chunk.limit",
       "hive.exec.orc.max.disk.range.chunk.limit",
     Integer.MAX_VALUE - 1024, "When reading stripes >2GB, specify max limit for the chunk size."),
+  ORC_MIN_DISK_SEEK_SIZE("orc.min.disk.seek.size",
+                         "hive.exec.orc.min.disk.seek.size",
+                         0,
+                         "When determining contiguous reads, gaps within this size are "
+                         + "read contiguously and not seeked. Default value of zero disables this "
+                         + "optimization"),
+  ORC_MIN_DISK_SEEK_SIZE_TOLERANCE("orc.min.disk.seek.size.tolerance",
+                          "hive.exec.orc.min.disk.seek.size.tolerance", 0.00,
```

Comment:
Can we remove this Hive conf? Actually, Apache Hive has no plan to use this, doesn't it?

Reply:
@dongjoon-hyun Agreed, I have reverted both the properties to use the ORC configuration